modules/mgmt-about-node-maintenance-schedules.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/mgmt-node-maintenance.adoc
+
+[id="mgmt-about-node-maintenance-schedules_{context}"]
+= About node maintenance schedules for bare metal nodes
+
+When you deploy {product-title} on bare metal infrastructure, there are additional considerations that must be taken into account compared to deploying on cloud infrastructure. Unlike in cloud environments where the cluster nodes are considered ephemeral, re-provisioning a bare metal node requires significantly more time and effort for maintenance tasks.
+
+When a bare metal node fails, for example, if the kernel hangs or a NIC card hardware failure occurs, workloads on the failed node need to be restarted elsewhere else on the cluster while the problem node is repaired or replaced. The Node Maintenance Operator allows cluster administrators to gracefully power down nodes, moving workloads to other parts of the cluster and ensuring workloads do not get interrupted. Detailed progress and node status details are provided and once the scheduled maintenance task is complete, the maintenance schedule task can simply be deleted, bringing the node back on-line.
```

Comment:
maybe seperate the "status" and "completed" parts in 2 sentences?
"Detailed progress and node status details are provided during maintenance. Once the scheduled maintenance task is complete, ..."